artist.php

```diff
@@ -1,3 +1,151 @@
 <?php
+declare(strict_types=1);
 
-declare(strict_types=1);
\ No newline at end of file
+$artist = (int)$_GET['artist'];
```

Comment:
You should check this with isset. If someone navigates to this page without ?artist=x in the URL, just redirect them back to index.php I think

Reply:
Also, the link to this page from index.php is broken - it just has ?1 in the URL